README.md

```diff
@@ -58,10 +61,13 @@ Now in another terminal, go back to the `scratch-gui` folder and run
 ```bash
 npm install
 npm link scratch-paint
+npm run build
```

Comment:
You're seeing npm run build is necessary in scratch gui too?

Reply:
ooohhhh no, thanks for catching this